pkg/oc/cli/admin/release/info.go

```diff
@@ -540,7 +540,9 @@ func (o *InfoOptions) LoadReleaseInfo(image string, retrieveImages bool) (*Relea
 			SecurityOptions: o.SecurityOptions,
 			ParallelOptions: o.ParallelOptions,
 			ImageMetadataCallback: func(name string, image *imageinfo.Image, err error) error {
-				verifier.Verify(image.Digest, image.ContentDigest)
+				if image != nil {
```

Comment:
Why not changing the order, I mean checking for error first and only then for image?

Reply:
being consistent about calling verify first so I can audit it later.

Reply:
in this case being able to check that we call Verify everywhere is the most important